superset-frontend/stylesheets/less/variables.less

```diff
@@ -47,7 +47,7 @@
 
 @almost-black: #263238;
 @gray-dark: #484848;
-@gray-light: #cfd8dc;
+@gray-light: #e0e0e0;
```

Comment:
This is going to affect (apparently) at least 68 other things in the codebase. Is this a situation where we could use Emotion's `styled` to apply this color in a more specific place (using `theme.colors.grayscale.light2`)?

Reply:
Actually that was the intention because `#e0e0e0` is the official gray-light theme color.